Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs

```diff
@@ -1723,29 +1723,231 @@ public void When_non_browsable_property_differs_excluding_non_browsable_members_
             subject.Should().BeEquivalentTo(expectation, config => config.ExcludingNonBrowsableMembers());
         }
 
+        [Fact]
+        public void When_property_is_non_browsable_only_in_subject_excluding_non_browsable_members_should_make_it_succeed()
+        {
+            // Arrange
+            var subject = new ClassWithNonBrowsableMembers() { NonBrowsableProperty = 0 };
+            var expectation = new ClassWithMatchingMemberNamesButEverythingIsBrowsable() { NonBrowsableProperty = 1 };
```

Comment:
🤔 This confuses me. The property is called `NonBrowsableProperty`, but it is still browsable.

Reply:
I thought `Class With Matching Member Names But Everything Is Browsable` would explain it, but I can make make data types specific to that test.

Reply:
It's not a critical issue, but I'm pretty sure it confuses us again if we look at it in 6 months.

Reply:
I reworked those tests so that the member name is now `PropertyThatMightBeNonBrowsable` and the type name is very clear about whether it does or does not actually mark it non-browsable.